inst/private/python_ipc_popen2.m

```diff
@@ -43,24 +43,11 @@
   info = [];
 
   if (strcmp(what, 'reset'))
-    if (~isempty(pid))
-      if (verbose)
-        disp ('Closing the Python communications link.')
-      end
-    end
-    if (~isempty(fin))
-      % produces a single newline char: not sure why
-      t = fclose(fin); fin = [];
-      waitpid(pid);
-      pid = [];
-      A = (t == 0);
-    else
-      A = true;
-    end
-    if (~isempty(fout))
-      t = fclose(fout); fout = [];
-      A = A && (t == 0);
-    end
+    cleanup = [];
+    fin = [];
+    fout = [];
+    pid = [];
+    A = true;
```

Comment:
Unfortunately I can't figure out a way to both use `onCleanup` and capture the return value